Scripts/Mac/LayoutFunctionalTests.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
We had discussed reorganizing the Windows scripts into dev-oriented and CI-oriented scripts. This one very clearly feels like a script intended for use on the CI/PR builds, and never on a dev box, so we could we stick it in a different location and not mix it in with the dev scripts?

Reply:
Makes sense, I'll move it.